sphinx/ext/autosummary/__init__.py

```diff
@@ -643,7 +643,9 @@ def _import_by_name(name: str) -> Tuple[Any, Any, str]:
             try:
                 mod = import_module(modname)
                 return getattr(mod, name_parts[-1]), mod, modname
-            except (ImportError, IndexError, AttributeError):
+            except (ImportError, IndexError, AttributeError) as e:
+                if last_errors is not None:
+                    last_errors.append(str(str(e.args[0])))
```

Comment:
```suggestion
                    last_errors.append(str(e))
```

Don’t stringify twice. Also you can just rely on the builtin `__str__` method that also accesses `self.args[0]` but actually has a fallback in case `len(self.args) == 0` instead of crashing

Reply:
Oh right, copy pasta error. Thanks!